gitignore

```diff
@@ -1,21 +1,49 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-if [ -z "${1:-}" ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
-    echo -e "Generate .gitignore file for the language you prefer.\nUses https://gitignore.io\n"
+# colors
+NC='\033[0m' # No Color
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+
+
+if [ -z "${1:-}" ] || [[ ${@} =~ "," ]] || printf '%s\n' ${@} | grep -q -P '^--?h(elp)?$'
+then
+    echo "Generate .gitignore file for the language you prefer."
+    echo -e "Uses https://gitignore.io\n"
     echo "Usage:"
-    echo -e "  gitignore <LANGUAGE>\n"
+    echo -e "  gitignore <environments> \n"
     echo "For example:"
-    echo "  gitignore python"
+    echo "  gitignore macos python pycharm"
```

Comment:
let's also include one example in help that only contains language too.

Reply:
would the inclusion of examples of README work ?

Reply:
that's exactly what I am talking about. That would be fine